modules/nodes-nodes-viewing-listing.adoc

```diff
@@ -171,9 +199,46 @@ Events:     <11>
 <3> The labels applied to the node.
 <4> The annotations applied to the node.
 <5> The taints applied to the node.
-<6> Node conditions.
+<6> The node conditions and status. The `conditions` stanza lists the `Ready`, 'PIDPressure`, 'PIDPressure`, 'MemoryPressure`, 'DiskPressure` and `OutOfDisk` status. These condition are described later in this section.
 <7> The IP address and host name of the node.
 <8> The pod resources and allocatable resources.
 <9> Information about the node host.
 <10> The pods on the node.
-<11> The events reported by the node.
+<11> The events reported by the node.   
+
+Among the information shown for nodes, the following node conditions appear in the output of the commands shown in this section:
+
+[discrete]
+[id="machine-health-checks-resource-conditions"]
+.Node Conditions
+[cols="3a,8a",options="header"]
+|===
+
+|Condition |Description
+
+|`Ready`
+|If `true` the node is healthy and ready to accept pods. If `false` the node is not healthy and is not accepting pods. If `unknown`, the node controller has not received a heartbeat from the node for the `node-monitor-grace-period` (the default is 40 seconds).
```

Comment:
(Comment holds for all of these) - should have comma after "If `true`" and similar.